yarn.lock

```diff
@@ -1756,6 +1756,16 @@
     esquery "^1.4.0"
```

Comment:
If it's not a major effort.
It could be nice to unify this on to npm and npm-workspaces.

Reply:
I personally prefer npm over yarn nowadays too (yarn 1 is barely maintained anymore). However, I consider this out of scope for this PR. I’m just working with the tools available. :)